spec/example_app/db/seeds.rb

```diff
@@ -10,12 +10,20 @@
 Order.destroy_all
 Customer.destroy_all
 Product.destroy_all
+Country.destroy_all
+
+Country.create(code: "US", name: "USA")
+Country.create(code: "CA", name: "Canada")
+Country.create(code: "CN", name: "China")
+Country.create(code: "RU", name: "Russia")
+Country.create(code: "AU", name: "Australia")
```

Comment:
Ok, good point, but should we also add create-bang to other models in seeds.rb, i did `create` to preserve uniformity. Also i'll better stick with old definition of US 🤣